ui/app/common/photo-capture/directives/capturePhoto.js

```diff
@@ -138,11 +138,11 @@ angular.module('bahmni.common.photoCapture')
                     var fileType = this.files[0].type;
                     if (!fileType.startsWith('image/')) {
                         uploadConfirmImageButton.prop('disabled', true);
-                        alert('File uploaded must be an image');
+                        alert($translate.instant("FILE_UPLOAD_MUST_BE_IMAGE"));
```

Comment:
These are existing keys in translation files?

Reply:
No added new ones.

Reply:
Should the translation file also be in this PR then?

Reply:
I have added it in clinic-config repo and raised PR there.

Reply:
> Should the translation file also be in this PR then?

Added translation file in bahmniapps too.